File: PetPal-Business/Helpers/UserParams.cs
namespace PetPal_Business.Helpers
{
    public class UserParams : PaginationParams
    {
        public string? CurrentUsername { get; set; }
        public string? Mood { get; set; }
        public int MinAge { get; set; } = 1;
        public int MaxAge { get; set; } = 100; //change this so it makes sense in "dog years". 1-14 ish.
        public string OrderBy { get; set; } = "lastActive";

    }
}

File: PetPal-Client/src/app/_models/userParams.ts
import { User } from "./user";

export class UserParams {
    mood: string;
    minAge = 1;
    maxAge = 14;
    pageNumber = 1;
    pageSize = 6
    orderBy = 'lastActive';

    constructor(user: User) {
        this.mood = user.mood === 'playful' ? 'playful' : 'calm'
    }
}